### PR TITLE
CWS: scheduled sync of btfhub constants

### PR DESCRIPTION
**Pull request by paulcacheux**
### What does this PR do?

This PR automatize a repetitive task done by the CWS of syncing with BTFHub constants.
This new workflow will create a PR with needed changes if needed. The approval/merge will still be done manually.

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by spikat — APPROVED** [review]

File: .github/workflows/cws-btfhub-sync.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      - name: Install python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8.10'
+          cache: 'pip'
+
+      - name: Install go
+        uses: actions/setup-go@v3
+        with:
+          go-version: '1.17.11'
```
Comment (bkabrda) [inline .github/workflows/cws-btfhub-sync.yml:32]: Should we wait for https://github.com/DataDog/datadog-agent/pull/13197 to get merged and then use `.go-version` here?

File: .github/workflows/cws-btfhub-sync.yml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+          ref: main
+
+      - name: Checkout btfhub-archive repository
+        uses: actions/checkout@v3
+        with:
+          repository: aquasecurity/btfhub-archive
+          path: dev/dist/archive
+
+      - name: Install python
+        uses: actions/setup-python@v2
```
Comment (bkabrda) [inline .github/workflows/cws-btfhub-sync.yml:24]: I think the latest version is `v4`?

**Review by bkabrda — CHANGES_REQUESTED** [review]
👋 the PR mostly looks good, but there are 2 minor things that I commented on that I'd like to see addressed.

File: .github/workflows/cws-btfhub-sync.yml (hunk 1 above)
Comment (paulcacheux) [reply .github/workflows/cws-btfhub-sync.yml:32]: Absolutely we can

File: .github/workflows/cws-btfhub-sync.yml (hunk 2 above)
Comment (paulcacheux) [reply .github/workflows/cws-btfhub-sync.yml]: good point, and fixed in last commit. Thanks !

File: .github/workflows/cws-btfhub-sync.yml (hunk 1 above)
Comment (paulcacheux) [reply .github/workflows/cws-btfhub-sync.yml]: Should be good now

**Review by bkabrda — APPROVED** [review]

**Pull request merged**